openapi_python_client/templates/model.pyi

```diff
@@ -38,13 +38,18 @@ class {{ model.reference.class_name }}:
     {% endif %}
 
 
-    def to_dict(self) -> Dict[str, Any]:
+    def to_dict(self, _serialize_options: Dict[str, Any]={}) -> Dict[str, Any]:
```

Comment:
`to_dict` is the method that every model class has for encoding to a JSON-serializable data structure.

The new information we need to know is: do we need to skip read-only properties? Instead of adding an optional `_skip_read_only: bool=False`, I chose to make it more open-ended and pass a dict. I think it just makes it a little conceptually clearer, when this object is being passed around in other templates, that it represents "stuff we need to know when serializing" in general.